### ISSUE TITLE
Mass Unprivater as an option to Mass Privater

### ISSUE THREAD

**Issue by LadyHaywood** [AprilSylph/XKit-Rewritten#1626]
### Concept

In the same way how you can mass private post based on tags and time frame, I would like to unprivate post based on the same things instead of going through anything one by one.

**Comment by marcustyphoon** [comment]
We'll definitely add this if Tumblr's Mass Post Editor (the back end API of which we use here) gets an unprivate feature, or at least if its back end does! Send a suggestion in to https://www.tumblr.com/support if you'd like that. If not, the only way we could make a mass unprivate feature would process one post per second instead of a hundred, which is of course way slower.

Though I suppose it would still be a useful feature no matter how slow it was—one might have few posts or be fine leaving it to run overnight, and there's no other way to filter private posts by tag at all, so it's not like there's an alternative.

**Comment by LadyHaywood** [comment]
Yeah, I didn't realize that private post aren't technically publushed yet until recently so selecting them by wouldn't really work in the same way mass privater works. Having that option in the mass post editor would be nice even if it'll work through it one by one. As long as I don't have to be sitting there doing it myself~.
